test/vstest/runtests.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --version $2 --channel 2.0 --no-cdn --install-dir $HELIX_CORRELATION_PAYLOAD/sdk
+export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
```

Comment:
Just thought of a few more environment variables we should probably set in Linux **and** Windows.
```sh
# Ensures every invocation of dotnet apps uses the same dotnet.exe
export DOTNET_ROOT="$HELIX_CORRELATION_PAYLOAD/sdk"

# Ensure dotnet comes first on PATH
export PATH="$DOTNET_ROOT:$PATH"

# Prevent fallback to global .NET locations. This ensures our tests use the shared frameworks we specify and don't rollforward to something else that might be installed on the machine
export DOTNET_MULTILEVEL_LOOKUP=0

# Avoid contaminating userprofiles
export DOTNET_CLI_HOME="$HELIX_CORRELATION_PAYLOAD/home"
```